### PR TITLE
Add RBAC desc and fix format for Dashboard user management

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Fix format
- Add RBAC authorize example desc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
I've finished the self-check of this PR. PTAL

File: dashboard/dashboard-user.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ TiDB Dashboard 与 TiDB 使用相同的用户权限体系和登录验证方式
 
 ## 示例：通过 RBAC 授权 SQL 用户登录 TiDB Dashboard
 
-以下示例演示了如何在[基于角色的访问控制（RBAC）](/role-based-access-control.md)机制下创建角色及用户来登录 TiDB Dashboard。
+以下示例演示了如何在[基于角色的访问控制 (RBAC)](/role-based-access-control.md)机制下创建角色及用户来登录 TiDB Dashboard。
```
Comment (ran-huang) [inline dashboard/dashboard-user.md:74]:
```suggestion
以下示例演示了如何在[基于角色的访问控制 (RBAC)](/role-based-access-control.md) 机制下创建角色及用户来登录 TiDB Dashboard。
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: dashboard/dashboard-user.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ TiDB Dashboard 与 TiDB 使用相同的用户权限体系和登录验证方式
     - SYSTEM_VARIABLES_ADMIN
 
 > **注意：**
-> 
-> 拥有 `ALL PRIVILEGES` 或 `SUPER` 等粗粒度高权限的用户同样可以登录 TiDB Dashboard。出于最小权限原则，强烈建议创建用户时仅使用上述精细权限，从而防止用户执行非预期操作。请参阅[权限管理](/privilege-management.md)了解这些权限的详细信息。
+>
+> 拥有 `ALL PRIVILEGES` 或 `SUPER` 等粗粒度高权限的用户同样可以登录 TiDB Dashboard。出于最小权限原则，强烈建议创建用户时仅使用上述精细权限，从而防止用户执行非预期操作。请参阅[权限管理](/privilege-management.md) 了解这些权限的详细信息。
```
Comment (TomShawn) [inline dashboard/dashboard-user.md:37]: This space is not needed. The markdown format is rendered as "请参阅[权限管理](/privilege-management.md)了解这些权限的详细信息。"
```suggestion
> 拥有 `ALL PRIVILEGES` 或 `SUPER` 等粗粒度高权限的用户同样可以登录 TiDB Dashboard。出于最小权限原则，强烈建议创建用户时仅使用上述精细权限，从而防止用户执行非预期操作。请参阅[权限管理](/privilege-management.md)了解这些权限的详细信息。
```

**Pull request merged**